### PR TITLE
Clean up internationalization

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: projects/epsilon-blueprint/src/lib/table/models/column.ts [hunk 1]
```diff
@@ -31,14 +31,8 @@ export interface Column {
...
 export enum ColumnType {
-  ID,
-  TEXT,
+  ICON,
   LINK,
-  DATE,
-  BOOLEAN,
-  INPUT,
   STATUS,
   TEMPLATE,
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/models/column.ts]: Can we delete the trailing comma? :)

File: projects/epsilon-blueprint/src/lib/table/table-footer/display-density/display-density.component.ts [hunk 2]
```diff
@@ -11,22 +11,30 @@ export class DisplayDensityComponent implements OnInit {
...
+      value: option
+    }));
     this.displayDensity = !localStorage.getItem('selectedDensity')
-      ? this.internationalization['Display Density Options'][0]
+      ? this.displayDensity
       : localStorage.getItem('selectedDensity');
   }
 
   setDisplayDensity() {
+    console.log(this.displayDensity);
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/table-footer/display-density/display-density.component.ts]: Delete console.log

File: projects/epsilon-blueprint/src/lib/table/table.component.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ export class TableComponent implements OnInit, OnChanges {
   @Output() action = new EventEmitter();
   @Output() selectedRowsAction = new EventEmitter();
 
+  // Defaults
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/table/table.component.ts]: Do we really need this comment?

File: src/app/modules/component-library/library/table/models/column.ts [hunk 4]
```diff
@@ -31,14 +31,8 @@ export interface Column {
...
 export enum ColumnType {
-  ID,
-  TEXT,
+  ICON,
   LINK,
-  DATE,
-  BOOLEAN,
-  INPUT,
   STATUS,
   TEMPLATE,
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/table/models/column.ts]: Trailing comma? :)

File: src/app/modules/component-library/library/table/overview/overview.component.ts [hunk 5]
```diff
@@ -27,7 +27,7 @@ export class OverviewComponent implements OnInit {
 
   ngOnInit() {    
     this.properties = {
-      internationalization: {
+      /* internationalization: {
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/table/overview/overview.component.ts]: Delete commented out code

File: projects/epsilon-blueprint/src/lib/table/table.component.ts (hunk 3 above)
Comment (APJohns) [reply projects/epsilon-blueprint/src/lib/table/table.component.ts]: The rest of the variable groups are commented, helps know where you are in the code.

**Pull request merged**